authenticate-users/test/FunctionsTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use PHPUnit\Framework\TestCase;
```

Comment:
Looks like we need to require phpunit

Reply:
I'm installing `phpunit/phpunit:7` in [.travis.yaml](https://github.com/GoogleCloudPlatform/getting-started-php/blob/master/.travis.yml#L41) globally and using this for all tests. It's not as straightforward but it's more efficient, and keeps the `composer.json` cleaner, and I don't think many users are going to be running the tests.